content/en/blog/_posts/2022-10-05-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog
+title: "Current State: 2019 Third Party Security Audit of Kubernetes"
+date: 2022-10-05
+slug: current-state-2019-third-party-audit
+---
```

Comment:
```suggestion
evergreen: true
---
```
This article will not need updates as it is an accurate point in time snapshot.
Although, I would suggest putting the month and year into the `title` to avoid a small risk of folks getting misled.

Reply:
(this is an optional improvement)